forte/data/ontology/validation_schema.json

```diff
@@ -95,6 +95,14 @@
                 "value_type": {
                   "description": "Item type for the case of Dice attributes",
                   "type": "string"
+                },
+                "ndarray_dtype": {
```

Comment:
do we have some tests to validate a JSON spec with this schema, hopefully, both success and failure cases?

Reply:
Added a test against invalid shape.
Do you think we can rely on numpy to validate dtype?

Reply:
it'd be great if schema can check the possible dtype values. Since this is done without actually running the code, while numpy validation happens late at run time.

Reply:
sounds good.
I've added more tests against dtype and shape

Reply:
the validation looks awesome, and thanks for the tests